### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by te-sa**
- added comments to classes
- added to README

Let me know if you think everything is documented clearly enough, and comment on any code that you feel could use more explanation.

### CODE REVIEW

**Review by DenisGauthier — APPROVED** [review]
Looks good! Thanks for cleaning it up and explaining!

File: app/src/main/res/layout/activity_edit_todo.xml [hunk 1]
```diff
@@ -77,10 +79,6 @@
             android:layout_height="wrap_content"
             app:layout_constraintBottom_toBottomOf="parent">
 
-            <LinearLayout
-                android:layout_width="match_parent"
-                android:layout_height="70dp"
-                android:orientation="horizontal"></LinearLayout>
```
Comment (justino599) [inline app/src/main/res/layout/activity_edit_todo.xml:83]: This piece of code is necessary for making sure there is enough space for the error messages to appear, specifically that 70dp of height. Without this, when an error appears, the buttons move down the screen to accommodate the message. If you are okay with that behaviour, you can leave it out, but you may want to leave it in.

**Review by justino599 — COMMENTED** [review]
Just put that one comment on there. Up to you if you want to change it.

File: app/src/main/res/layout/activity_edit_todo.xml (hunk 1 above)
Comment (te-sa) [reply app/src/main/res/layout/activity_edit_todo.xml:83]: Which error message do you mean? The "Please enter a task name" one? I'll probably add the code back in just to be safe

Comment (justino599) [reply app/src/main/res/layout/activity_edit_todo.xml:83]: When you're editing a task and try to leave the name blank, or if you enter an invalid date, there is a corresponding Snackbar message that appears.

Comment (te-sa) [reply app/src/main/res/layout/activity_edit_todo.xml]: Ok, I put that back in. Thx for explaining, the screenshots were helpful

**Review by justino599 — APPROVED** [review]

**Pull request merged**